components/tree-select/index.en-US.md

```diff
@@ -0,0 +1,56 @@
+---
+category: Components
+type: Form Controls
+title: TreeSelect
+---
+
+Tree selection control.
+
+## When to use
+
+`TreeSelect` is a selection control like `Select`, which has a tree shaped data structure, such as corporate hierarchy、 disciplinary system、 directory structure and etc.
```

Comment:
`TreeSelect` is similar with `Select`, but the values are provided in a tree like structure.
Any data whose entries are defined in a hierarchical manner is fit to use this control. Examples of such case may include a corporate hierarchy, a directory structure, and so on.